### PR TITLE
chore(#40): depracate DB_ADMIN_XXX in favor of DB_XXX to aling with

### PR DESCRIPTION
**Pull request by iocanel**
the rest rdbms playbooks.

### CODE REVIEW

**Comment by tchughesiv** [comment]
@iocanel tks for the contribution. with the other RDBMS offerings, DB_USER is leveraged for unprivileged user accounts. But, w/ mysql for example, there are separate variables for root/admin access.

e.g. https://github.com/ansibleplaybookbundle/mariadb-apb/blob/83eb0ff0026fa0485fb5b0bd148ffa9266d46bf2/templates/deployment.yaml.j2#L54

w/ mssql, the SA account is that equivalent so it feels as if we should reserve the DB_USER variable for a regular user. this "user create" logic is not yet built into the mssql image or apb like it is w/ the other RDBMS offerings. some modifications would first have to be made to the image and apb before switching to a DB_USER var.

**Comment by iocanel** [comment]
So what do u suggest? Remove the deprecation comment and keep both sets around until support for admin password is added?

**Comment by iocanel** [comment]
fwiw, I want to align books as much as possible so that I caneasier maintain: https://github.com/snowdrop/servicecatalog-connector

File: roles/provision-mssql-apb/tasks/main.yml [hunk 1]
```diff
@@ -239,6 +239,9 @@
       DB_TYPE: mssql
       DB_HOST: '{{ service_instance }}.{{ namespace }}.svc'
       DB_PORT: '{{ service_port }}'
+      DB_USER: SA
+      DB_PASSWORD: '{{ mssql_sa_pw }}'
+      # deprecated: DB_ADMIN_USER and DB_ADMIN_PASSWORD will be replaced by DB_USER and DB_PASSWORD.
```
Comment (tchughesiv) [inline roles/provision-mssql-apb/tasks/main.yml:244]: @iocanel let's just remove this depreciation comment for now as you suggested.

**Review by tchughesiv — COMMENTED** [review]
@iocanel my only concern w/ this approach is that folks may get confused and think they're using an unprivileged user... but I think I'm ok w/ it until additional logic is added to offer separate users types.

**Comment by iocanel** [comment]
This is stale for way to long, closing.

**Pull request closed without merging**